### PR TITLE
Booking component

### PR DESCRIPTION
**Pull request by SelmaNdapanda**
### In this pull request, I implemented the following:  :arrow_down: 

- When the user clicks the "Booking" link in the navigation panel they can see a form for adding a new item.  :heavy_check_mark: 
- Set up the routes in the App component and create the booking component.  :heavy_check_mark: 
- Make sure that only authorized users can make a reservation/booking.  :heavy_check_mark:

### CODE REVIEW

**Review by yonas44 — APPROVED** [review]
## STATUS: APPROVED :heavy_check_mark: 
Hi @SelmaNdapanda  , 

To highlight 👍🏽 
✔️ Clean code
✔️ Gitflow
✔️ No errors


Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

### Optional suggestions: please check below

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/bookings/bookings.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+export default function Bookings() {
+  const isAuthenticated = getToken;
+
+  if (!isAuthenticated) {
+    return <Navigate to="/sign_in" />;
+  }
+
+  return <BookingForm />;
+}
```
Comment (yonas44) [inline src/components/bookings/bookings.js:102]: Nice handling redirecting unsigned user, but this code doesn't seem to help the user be navigated to the sign in page when they are not, rather the reservations page helps with the redirecting, so add a proper way to handle redirecting or just remove it, as it has no purpose at the moment.

Hint: One way to solve it is to add a condition in a `useEffect` which checks if there is a token present in the sessionStore

File: src/redux/reservations/postReservations.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: getToken(),
+      },
+      body: JSON.stringify({ booking: object }),
+    });
+  },
+);
+
+export default postReservations;
```
Comment (yonas44) [inline src/redux/reservations/postReservations.js:18]: This request is not properly handled in the slice, for the next time make sure to add cases to handle if the request is fulfilled, pending and rejected :+1:

File: src/components/bookings/bookings.js (hunk 1 above)
Comment (SelmaNdapanda) [reply src/components/bookings/bookings.js:102]: Thank you for your review. I will implement this :+1:

File: src/redux/reservations/postReservations.js (hunk 2 above)
Comment (SelmaNdapanda) [reply src/redux/reservations/postReservations.js:18]: Okay. Thank you for the heads up. I will implement it :+1:

**Pull request merged**